data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -55,6 +56,7 @@ then
 
 	bootkube_podman_run \
 		--volume "$PWD:/assets:z" \
+		--env CLUSTER_PROFILE="{{.ClusterProfile}}" \
```

Comment:
Since this is still somewhat experimental, let's omit the environment variable in the CVO pod unless the user has set the cluster profile.

Reply:
changed